spec/unit/client/redis/apns2/notification_spec.rb

```diff
@@ -0,0 +1,47 @@
+# encoding: US-ASCII
+
+require "unit_spec_helper"
+
+describe Rpush::Client::Redis::Apns2::Notification do
+  it_behaves_like 'Rpush::Client::Apns::Notification'
+
+  it "should validate the length of the binary conversion of the notification" do
+    notification = described_class.new
+    notification.device_token = "a" * 108
+
+    notification.alert = "a" * 4095
+    expect(notification.errors[:base]).to be_empty
+
+    notification.alert = "a" * 4096
```

Comment:
Unless I'm missing something, isn't 4096 still allowed and 4097 would be a problem? From the documentation:

> For regular remote notifications, the maximum size is 4KB (4096 bytes)

For me that reads as ≤4096.

Reply:
You're right, sorry, fix on the way.

Reply:
Pointing out the error was much appreciated, it highlighted that the test was utterly deficient. It should now be correct.